clap_derive/src/attrs.rs

```diff
@@ -308,6 +308,8 @@ impl Attrs {
                     self.push_method(ident, self.name.clone().translate(*self.env_casing));
                 }
 
+                Hidden(ident) => self.push_method(ident, true),
```

Comment:
Is there a reason you added support for `#[clap(hidden)]` on top of the existing `#[clap(hidden=true)]`?

This means we are being inconsistent with our bool methods.  Seems like we should do them all or none (unless there is a specific motivation for a subset)

Reply:
There is no paticular reason for this, I tried to match the surrounding code, as this is my first time doing derive macros.